src/applications/post-911-gib-status/actions/post-911-gib-status.js

```diff
@@ -31,7 +32,11 @@ export function getEnrollmentData(apiVersion) {
         const error =
           response.errors.length > 0 ? response.errors[0] : undefined;
         if (error) {
-          if (error.status === '503' || error.status === '504') {
+          if (error.status === '503') {
+            // LTS Service Downtime Error
+            return dispatch({ type: SERVICE_DOWNTIME_ERROR });
```

Comment:
I'm not sure if that `DowntimeNotification` component is actually working, because I see requests to the endpoint in the Datadog dashboard we're using to monitor the endpoint.

I added new messaging to specifically display the downtime information